ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -115,6 +115,11 @@ private class JdbcLedgerDao(
   override def lookupLedgerEnd()(implicit loggingContext: LoggingContext): Future[Offset] =
     dbDispatcher.executeSql(metrics.daml.index.db.getLedgerEnd)(ParametersTable.getLedgerEnd)
 
+  // THIS METHOD WILL BE IMPLEMENTED WITH THE APPEND-ONLY SCHEMA WORK
```

Comment:
This method is not compatible with the legacy schema and it will not be made compatible. It's better to just throw an unsupported here

Reply:
Done.